scripts/west_commands/runners/core.py

```diff
@@ -649,6 +660,17 @@ def dev_id_help(cls) -> str:
                   target when multiple ones are available or
                   connected.'''
 
+    @classmethod
+    def extload_help(cls) -> str:
+        ''' Get the ArgParse help text for the --extload option.'''
+        return '''External loader to be used by stm32cubeprogrammer
+                  to program the targeted external memory.
+                  The runner requires the external loader (*.stldr) filename.
+                  This external loader (*.stldr) must be located within
+                  STM32CubeProgrammer/bin/ExternalLoader directory.
+                  STM32_Programmer_CLI 's -el expects and retrieves
+                  the absolute path for this external loader'''
```

Comment:
```suggestion
```

That info is not necessary here. It was provided in a comment to highlight the existing difference.
When people use the cli tool they should refer to the tool's manual for such information.

Reply:
@FRASTM can you take this one into account ?

Reply:
Edit: Request is to remove the non necessary information. Then I propose to focus current PR to use tools loader. Extending to custom out-of-tree loaders could be added later.